modules/xmpp/xmpp.js

```diff
@@ -209,9 +209,9 @@ export default class XMPP extends Listenable {
         this.caps.addFeature('urn:xmpp:jingle:apps:rtp:video');
         this.caps.addFeature('http://jitsi.org/json-encoded-sources');
 
-        // Disable RTX on Firefox 83 and older versions because of
-        // https://bugzilla.mozilla.org/show_bug.cgi?id=1668028
-        if (!(this.options.disableRtx || (browser.isFirefox() && browser.isVersionLessThan(94)))) {
+        // Disable RTX on Firefox because we prefer simulcast over RTX
```

Comment:
Suggestion: bring back `browser.supportsRtx()` to avoid too many browser specific checks throughout the code.